pkg/jaeger/query/trace_query.go

```diff
@@ -181,8 +181,9 @@ func buildTraceIDSubquery(q *spanstore.TraceQueryParameters) (string, []interfac
 
 	if len(q.Tags) > 0 {
 		for k, v := range q.Tags {
-			params = append(params, k, v)
-			qual := fmt.Sprintf(`s.span_tags ? ($%d == $%d)`, len(params)-1, len(params))
+			//TODO make sure this is optimized correctly
```

Comment:
The following seems to work in PSQL. Should be good to go. I'll double-check with `pgx` driver and we can fix post-RC if there's an issue. 

```SQL
/* Equals operator returns correct result */
PREPARE eq_test AS
SELECT trace_id
    FROM ps_trace.span
    WHERE
            span_tags -> $1 = $2::text::jsonb
        AND resource_tags -> $3 = $4;

EXECUTE eq_test('pwlen', 25, 'service.name', '"generator"');

/* Equals uses support function and produces correct plan */
SELECT 
    x::text LIKE '%InitPlan%',
    x::text LIKE '% @> %' 
FROM explain_jsonb($$EXECUTE eq_test('pwlen', 25, 'service.name', '"generator"')$$) AS f(x);
```